ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -57,6 +57,13 @@ tests:
       COLLECTION_METHOD: core_bpf
     workflow: stackrox-stackrox-e2e-job
   timeout: 5h0m0s
+- always_run: false
+  as: ui-component-tests
+  optional: true
+  run_if_changed: ^ui/
```

Comment:
Can you remove this 'for now'. IIUC `always_run: false` has precedence, so this job should not run on any PRs unless explicitly requested but I'm never 100% sure what prow will do.
```suggestion
```